WeatherSystem.Test/UserShould.cs

```diff
@@ -0,0 +1,83 @@
+﻿using WeatherSystem.Strategies;
+
+namespace WeatherSystem.Test
+{
+
+    public class UserShould
+    {
+        [Theory]
+        [InlineData("Nablus" , 32 , 40.1)]
+        public void Parse_JSON_Reurtn_Same_Location(string actualLocation , float actualTempreture , float actualHumidity)
+        {
+            var weatherData = $"{{\"Location\": \"{actualLocation}\", \"Temperature\": {actualTempreture}, \"Humidity\": {actualHumidity}}}";
+            var context = new Context(new JsonReader());
+
+            var data = context.ExecuteStrtegy(weatherData);
+
+           Assert.Equal(data.Location , actualLocation);
+        }
+
+
+        [Theory]
+        [InlineData("Nablus", 32, 40.1)]
+        public void Parse_JSON_Reurtn_Same_Tempereture(string actualLocation, float actualTempreture, float actualHumidity)
+        {
+            var weatherData = $"{{\"Location\": \"{actualLocation}\", \"Temperature\": {actualTempreture}, \"Humidity\": {actualHumidity}}}";
+            var context = new Context(new JsonReader());
+
+            var data = context.ExecuteStrtegy(weatherData);
+
+            Assert.Equal(data.Temperature, actualTempreture);
+        }
+
+
+        [Theory]
+        [InlineData("Nablus", 32, 40.1)]
+        public void Parse_JSON_Reurtn_Same_Humidity(string actualLocation, float actualTempreture, float actualHumidity)
+        {
+            var weatherData = $"{{\"Location\": \"{actualLocation}\", \"Temperature\": {actualTempreture}, \"Humidity\": {actualHumidity}}}";
+            var context = new Context(new JsonReader());
+
+            var data = context.ExecuteStrtegy(weatherData);
+
+            Assert.Equal(data.Humidity, actualHumidity);
```

Comment:
don't u need to assert other props in the object (temp and location) to make sure u got the exact object u expected?